sportran/current/current.py

```diff
@@ -236,8 +237,9 @@ def compute_psd(self, PSD_FILTER_W=None, freq_units='THz'):
         # number of degrees of freedom of the chi-square distribution of the psd / 2
         self.ndf_chi = self.N_EQUIV_COMPONENTS - self.N_CURRENTS + 1
         if self.ndf_chi <= 0:
-            raise RuntimeError('The number of degrees of freedom of the chi-squared distribution is <=0. The number of '
-                               'equivalent (Cartesian) components of the input current must be >= number of currents.')
+            warnings.warn('The number of degrees of freedom of the chi-squared distribution is <=0. The number of '
```

Comment:
@ppegolo this means that the bayesian filter works also when we have fewer independent time series than the number of components?